app/lib/config/development.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:munin/config/application.dart';
+
```

Comment:
Could we keep this out of version control? Since this file will be actively used during development, and if this is checked in, we'll have to manually exclude it from file list(or mask the credentials) everytime we want to commit.

I pushed an example config file for clarification https://github.com/edwardez/BangumiN/blob/develop/app/lib/config/ExampleEnvironment.dart

Reply:
Oh, It was used to trick CI and ensure the tree can be built after check out.
But as the app won't work without api credential, should we provide a way for developers in develpement build package to input their own credential?

Reply:
Seems like two discussions are going on at the same time.

> It was used to trick CI and ensure the tree can be built after check out

https://docs.github.com/en/actions/reference/encrypted-secrets seems like it's possible to configure credentials in github then let workflow read credentials from there


> should we provide a way for developers in develpement build package to input their own credential

I am assuming you are talking about developers manually compile the `.apk`(not about CI), we probably can add insturctions on how to create a local copy of `app/lib/config/development.dart` if someone want to build the package themselves

Reply:
Thanks. I'll drop CI part from this PR.

I'm just too lazy to setup flutter environment locally every time :-D

I was thinking about a way to let developer set credentials at both buildtime and runtime.